src/Microsoft.Performance.SDK.Runtime/CustomDataSourceReference.cs

```diff
@@ -14,9 +14,19 @@ namespace Microsoft.Performance.SDK.Runtime
     /// <summary>
     ///     Creates a <see cref="AssemblyTypeReferenceWithInstance{T, Derived}"/> where T is <see cref="ICustomDataSource"/> and Derived is <see cref="CustomDataSourceReference"/>.
     /// </summary>
-    public abstract class CustomDataSourceReference
+    public sealed class CustomDataSourceReference
```

Comment:
this is a breaking change; should we document it in the migrations/changes docs

Reply:
This is in the runtime. I don't believe that we note breaking changes for the runtime.

---
In reply to: [613291810](https://github.com/microsoft/microsoft-performance-toolkit-sdk/pull/73#discussion_r613291810) [](ancestors = 613291810)

Reply:
We recommend developers use the engine to access data, rather than directly accessing the runtime.

---
In reply to: [613293042](https://github.com/microsoft/microsoft-performance-toolkit-sdk/pull/73#discussion_r613293042) [](ancestors = 613293042,613291810)